main/src/library/pkger/pkger.flix

```diff
@@ -0,0 +1,43 @@
+pub enum SemVerComponent {
+    case Integer(Int),
+    case Wildcard,
+    case NotPresent
+}
+
+pub opaque type SemVer = { major: SemVerComponent
+                         , minor: SemVerComponent
+                         , patch: SemVerComponent
+                         }
+
+// Temporary
+pub opaque type SpecificSemVer = SemVer
+
+pub opaque type Package = { name: String
+                          , version: SpecificSemVer
+                          , authors: List[String]
+                          , description: String
+                          // This could be an enum. Would only support predefined licenses, or require an `Other` variant
+                          , license: Option[String]
```

Comment:
Yes, lets make it into an ADT with some predefined licenses and an Other(String)

Reply:
Will do!